docker-compose.yml

```
@@ -1,27 +1,12 @@
-version: '3.2'
+version: '3.7'
 
 services:
   dev:
     build:
       context: ./
       dockerfile: Dockerfile
-    entrypoint:
-      - "npm"
-    command: ["start"]
-    ports:
-      - "3735:3735"
-    environment:
-      - HOST=0.0.0.0
-      - AWS_REGION=${AWS_REGION}
-      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
-      - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
-      - AWS_SESSION_TOKEN=${AWS_SESSION_TOKEN}
-      - AWS_SECURITY_TOKEN=${AWS_SECURITY_TOKEN}
-      - PANOPTES_API_APPLICATION=${PANOPTES_API_APPLICATION}
-      - PANOPTES_API_HOST=${PANOPTES_API_HOST}
-      - STAT_HOST=${STAT_HOST}
-      - SUGAR_HOST=${SUGAR_HOST}
-      - TALK_HOST=${TALK_HOST}
+      args:
+        AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
+        AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
     volumes:
-      - ${PWD}:/src
       - /src/node_modules
```

With something like this you should be able to deploy by running `docker-compose build` followed by `docker-compose run deploy`.

```suggestion
       - /src/node_modules
  deploy:
    build:
      context: ./
      dockerfile: Dockerfile
    command: aws s3 sync /dist s3://clickworker-pfe.1715labs.com
    args:
      AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
      AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
```